packages/babel-preset/src/transformPlugin.ts

```diff
@@ -120,6 +122,7 @@ export const transformPlugin = declare<Partial<BabelPluginOptions>, PluginObj<Ba
         action: 'ignore',
       },
     ],
+    projectRoot: process.cwd(),
```

Comment:
is it safe to use cwd ? why not find the closest package.json ?

Reply:
Closest `package.json` is not really good option for monorepos as the same asset can be used in different packages:

```
/packages
/packages/assets
/packages/src-a # both "src-a" & "src-b" use a file from "assets", but relative path
/packages/src-b # will be different for "src-a" & "src-b" as both contain "package.json" files
```